vector/src/main/java/im/vector/app/features/spaces/SpaceSummaryController.kt

```diff
@@ -117,7 +123,7 @@ class SpaceSummaryController @Inject constructor(
                 ?.forEach { roomSummary ->
                     spaceSummaryItem {
                         avatarRenderer(host.avatarRenderer)
-                        id(roomSummary.roomId)
+                        id("invite_${roomSummary.roomId}")
```

Comment:
a unique id for the invite makes sense to me to fix the crash, we can iterate on the proper UX (whether two spaces with the same id should be present in the spaces list...)